### ISSUE TITLE
Pause/resume not functioning properly with v0.0.5

### ISSUE THREAD

**Issue by samwiseg0** [eliteSchwein/mooncord#195]
WIth version 0.0.5 pause and resume do not functioning properly. When the print is paused and then resumed it just states print started and the print job status is not updated there after. 

<img width="667" alt="Screen Shot 2022-01-08 at 2 02 54 PM" src="https://user-images.githubusercontent.com/2241731/148656615-928140dc-d2d3-44a0-b653-f689abd7943b.png">

Logs
```
[error] [2022-01-08T03:52:16.360Z] Error: Timed out after waiting for 20000 ms
            Error: Timed out after waiting for 20000 ms
    at /home/pi/mooncord/dist/index.js:7:849
    at async StatusHelper.update (/home/pi/mooncord/dist/index.js:49527:9)
    at async SubscriptionNotification.parsePrintStats (/home/pi/mooncord/dist/index.js:49835:13)
[info] [2022-01-08T04:43:35.128Z] klipper status changed to pause...

[error] [2022-01-08T04:43:55.131Z] Error: Timed out after waiting for 20000 ms
            Error: Timed out after waiting for 20000 ms
    at /home/pi/mooncord/dist/index.js:7:849
    at async StatusHelper.update (/home/pi/mooncord/dist/index.js:49527:9)
    at async SubscriptionNotification.parsePrintStats (/home/pi/mooncord/dist/index.js:49837:9)
[info] [2022-01-08T04:45:10.601Z] klipper status changed to start...

[error] [2022-01-08T04:45:30.603Z] Error: Timed out after waiting for 20000 ms
            Error: Timed out after waiting for 20000 ms
    at /home/pi/mooncord/dist/index.js:7:849
    at async StatusHelper.update (/home/pi/mooncord/dist/index.js:49527:9)
    at async SubscriptionNotification.parsePrintStats (/home/pi/mooncord/dist/index.js:49835:13)```

**Comment by samwiseg0** [comment]
As a note this error seems to always be present in the logs.  I have also configured periodic status updates and they are not firing. I am not sure if they are related or If I should open another issue just for that

```
[error] [2022-01-08T22:18:49.037Z] Error: Timed out after waiting for 20000 ms
            Error: Timed out after waiting for 20000 ms
    at /home/pi/mooncord/dist/index.js:7:849
    at async StatusHelper.update (/home/pi/mooncord/dist/index.js:49527:9)
 ```

**Comment by eliteSchwein** [comment]
i think i know where `Error: Timed out after waiting for 20000 ms` gets triggert

**Comment by eliteSchwein** [comment]
resume message is fixed with 355b855cb05ea368566cb802bdafef6e3cbd2b90
on the scheduler part: i will test it first, i think i forgot to implement it

**Comment by eliteSchwein** [comment]
scheduler fixed with 30051129fc5f0b34d775c9ddc492be9d20f42e3b
